congressional-support.yaml

```diff
@@ -0,0 +1,119 @@
+- type: house
+  name: House Documents Repository
+  url: http://docs.house.gov/
+
+- type: house
+  name: Office of Compliance
+  url: http://www.compliance.gov/
+
+- type: house
+  name: Clerk of the House
+  url: http://clerk.house.gov/
+
+- type: house
+  name: Chief Administrative Officer
+  url: http://cao.house.gov/
+
+- type: house
+  name: Sergeant at Arms
+  url:
+
+- type: house
+  name: Chaplain
+  url: http://chaplain.house.gov/
+
+- type: house
+  name: Inspector General
+  url: http://www.house.gov/content/learn/officers_and_organizations/inspector_general.php/
+
+- type: house
+  name: Historian
+  url: http://history.house.gov/
+
+- type: house
+  name: Law Revision Counsel
+  url: http://uscode.house.gov/
+
+- type: house
+  name: Legislative Counsel
+  url: http://legcounsel.house.gov/
+
+- type: house
+  name: Office of Congressional Ethics
+  url: http://oce.house.gov/
+
+- type: house
+  name: Office of the General Counsel
+  url: http://www.ogc.house.gov/
+
+- type: house
+  name: Office of Interparliamentary Affairs
+  url:
+
+- type: house
+  name: Parliamentarian
+  url:
+
+- type: house
+  name: Bipartisan Legal Advisory Group
+  url:
+
+- type: support agency
+  name: Library of Congress
+  url: http://www.loc.gov/
+
+- type: support agency
+  name: Government Publishing Office
+  url: http://www.gpo.gov/
+
+- type: support agency
+  name: General Accountability Office
+  url: http://www.gao.gov/
+
+- type: support agency
+  name: Capitol Police
+  url: http://www.uscapitolpolice.gov/
+
+- type: support agency
+  name: Architect of the Capitol
+  url: http://www.aoc.gov/
+
+- type: support agency
+  name: Congressional Budget Office
+  url: http://www.cbo.gov/
+
+- type: support agency
+  name: Congressional Research Service
+  url: http://www.loc.gov/crsinfo/
+
+- type: support agency other
```

Comment:
While they're in the legislative branch, they're not legislative support agencies. The copyright office, for example, does not provide support to the work of Congress, and for some purposes it is an executive branch agency. So, it gets an alternate designation.